save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/benchmarks/BenchmarkEntity.kt

```diff
@@ -19,6 +79,23 @@ enum class BenchmarkCategoryEnum {
     PERFORMANCE,
     STATIC_ANALYSIS,
     ;
+
+    companion object : MenuBar<BenchmarkCategoryEnum> {
+        override val defaultTab = ALL
+        val listOfStringEnumElements = BenchmarkCategoryEnum.values().map { it.name.lowercase() }
+        override val regex = Regex("/project/[^/]+/[^/]+/[^/]+")
```

Comment:
Maybe improve your regex specifying tab names?

Reply:
It is possible, but it will affect the logic further, which I would not want

Reply:
Done